tdm-be/tsconfig.json

```diff
@@ -1,14 +1,26 @@
 {
     "compilerOptions": {
-      "target": "es6",
-      "module": "commonjs",
-      "outDir": "./dist",
-      "strict": true,
-      "esModuleInterop": true,
-      "skipLibCheck": true,
-      "forceConsistentCasingInFileNames": true,
-      "resolveJsonModule": true
+        "target": "es6",
+        "module": "commonjs",
+        "lib": [
+            "ESNext"
+        ],
+        "outDir": "dist",
+        "strict": true,
+        "esModuleInterop": true,
+        "skipLibCheck": true,
+        "forceConsistentCasingInFileNames": true,
+        "resolveJsonModule": true,
+        "allowSyntheticDefaultImports": true,
+        "allowJs": true
     },
-    "include": ["**/*.ts"],
-    "exclude": ["node_modules", "dist"]
-  }
\ No newline at end of file
+    "include": [
+        "src/**/*.ts",
+        "src/**/*.js",
+        "config/config.json"
```

Comment:
<!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement.  -->
**todo:** Remove `config/config.json`

As it does not exist any more.‏